### PR TITLE
Automatic issue creation on anchore-scan failure

### PR DESCRIPTION
**Pull request by WSMathias**
[
Screenshot 2020-07-02 at 9 28 58 AM
](url)

### CODE REVIEW

File: .github/workflows/anchore.yml [hunk 1]
```diff
@@ -101,3 +113,14 @@ jobs:
         dockerfile-path: "php-fpm-apache/debian/Dockerfile"
         fail-build: false
         acs-report-enable: true
+    - name: Move JSON file
+      run: mv ./anchore-reports/vulnerabilities.json ./anchore-reports/scan_debian_php-fpm-apache.json
+    
+    # create issue
+    - name: Raise issue
+      id: raise_issue_php_dev
+      uses: WSMathias/create-an-issue-anchore@v1
```
Comment (AshishThakur) [inline .github/workflows/anchore.yml:122]: Lets create a bot user for this?

File: php-fpm-apache/debian/Readme.md [hunk 2]
```diff
@@ -5,15 +5,12 @@ CMD: start.sh [php-fpm and apache2]
 
 
 Aeguments:
-Name | Default value | description
+Name | Defailt value | description
```
Comment (AshishThakur) [inline php-fpm-apache/debian/Readme.md:8]: I see the the old typos which were fixed are pushed back.

**Comment by AshishThakur** [comment]
@WSMathias there are some unwanted changes in the Dockerfile..can you please check?

File: php-fpm-apache/debian/Readme.md (hunk 2 above)
Comment (WSMathias) [reply php-fpm-apache/debian/Readme.md]: Updated to only reflect the action changes

File: .github/workflows/anchore.yml (hunk 1 above)
Comment (AshishThakur) [reply .github/workflows/anchore.yml:122]: @WSMathias: bump. In case this is fixed..i will merge this.

Comment (WSMathias) [reply .github/workflows/anchore.yml]: done

**Pull request merged**